Update Tweets example in docs with new API (#534)
rotationalio/ensign: docs/content/examples/developers.md

	// ENSIGN_CLIENT_ID and ENSIGN_CLIENT_SECRET environment variables must be set
	var client *ensign.Client
	if client, err = ensign.New(); err != nil {
		panic("failed to create Ensign client: " + err.Error())
	}

> 	// Check to see if topic exists and create it if not

Reviewer: ```suggestion
	// Check to see if topic exists and create it if not
	exists, err := client.TopicExists(context.Background(), DistSysTweets)
	if err != nil {
		panic(fmt.Errorf("unable to check topic existence: %s", err))
	}
```

		panic("failed to create Ensign client: " + err.Error())
	}

	// Check to see if topic exists and create it if not
	exists, err := client.TopicExists(context.Background(), DistSysTweets)
	if err != nil {
		panic(fmt.Errorf("unable to check topic existence: %s", err))

> 	}

Reviewer 2: I think we still need this bit of code right?

> 		// The topic does exist, but we need to figure out what the Topic ID is, so we need
> 		// to query the ListTopics method to get back a list of all the topic nickname : topicID mappings

Reviewer 2: ```suggestion
		// The topic does exist, but we need the ID of the topic to publish to
```

)
```

> *Note that `client.Publish(topicID, e)` does not return an immediate error, it's an asynchronous operation so if we want to check for errors we have to do so after the fact. This means that we can't be sure which event actually triggered the error.*

Reviewer 2: ```suggestion
*Note that `client.Publish(topicID, e)` does not return an immediate error, it's an asynchronous operation so if we want to check if the event was actually published later on in the code we can use the `Acked()` or `Nacked()` methods on the event itself.*

```golang
acked, err := e.Acked()
if err != nil {
    fmt.Printf("could not check ack status for event: %s\n", err)
}

if acked {
    fmt.Println("event was committed!")
}

nacked, err := e.Nacked()
if nacked {
    fmt.Printf("event failed to commit with error: %s\n", err)
}
```
```
